Osucascades header

emerham opened the pull request.

emerham committed:
Added option for OSU Cascades header

emerham committed:
Applying code style updates

emerham committed:
Optimized the SVGs.

emerham commented:
Adds theme options only available to Admins and DX Administrators to be able to set the Companion Logo

Ruefa commented on logo.svg:
![image](https://github.com/osu-wams/madrone/assets/16888647/95ab093f-9672-4256-bf7f-a819183f7704)

emerham replied:
yeah the files were just "optimized". Will deal with the failing check on code standards

Ruefa reviewed (approved):
LGTM github actions already called you out

emerham committed:
Fix code standards.

emerham committed:
Testing a new action checker.

emerham committed:
attempting another check style

emerham committed:
Manually running phpcs instead of composer script

emerham committed:
Added custom script to run PHPCS with --checkstyle.
Update Github Action to use the new composer script and comment out a job that wasn't working with latest PHP features
